gradle/src/checkstyle.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <!DOCTYPE module PUBLIC
-    "-//Puppy Crawl//DTD Check Configuration 1.2//EN"
-    "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
+    "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+    "https://checkstyle.org/dtds/configuration_1_3.dtd">
```

Comment:
Is it necessary to change this DTD spec? This causes the checkstyle task to fail on my Windows workstation, since it doesn't seem to honor the SSL certificate of the `checkstyle.org` domain.

Reply:
That's weird. The original DTD URL seems to be a `404 not found` URL and that's why I changed it. I can change it back.